### PR TITLE
Update README to be more consistent with other SDKs

### PR DESCRIPTION
**Pull request by bidoubiwa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,23 +19,38 @@
...
+- [🔧 Installation](#-installation)
+- [🎬 Getting Started](#-getting-started)
+- [🤖 Compatibility with MeiliSearch](#-compatibility-with-meilisearch)
+- [💡 Learn More](#-learn-more)
+- [⚙️ Development Workflow and Contributing](#️-development-workflow-and-contributing)
+- [📜 Demos](#-demos)
+
+## 📖 Documentation
+
+For more information about this API see [our Swift documentation](https://meilisearch.github.io/meilisearch-swift/).
```
Comment (alallema) [inline README.md:40]:
```suggestion
For more information about this API see our [Swift documentation](https://meilisearch.github.io/meilisearch-swift/).
```
The next line is just the "documentation" and not "our documentation" in blue link, it's just a detail, but it triggers my eyes.

File: README.md [hunk 2]
```diff
@@ -19,23 +19,38 @@
...
+
+For more information about MeiliSearch see our [Documentation](https://docs.meilisearch.com/learn/tutorials/getting_started.html) or our [API References](https://docs.meilisearch.com/reference/api/).
+
+## 🔧 Installation
+
+### With Cocoapods <!-- omit in toc -->
 
 [CocoaPods](https://cocoapods.org/) is a dependency manager for Cocoa projects.
 
 **MeiliSearch-Swift** is available through [CocoaPods](https://cocoapods.org). To install
```
Comment (alallema) [inline README.md:50]: It's the exact same link in almost the same sentence may be just one it's enough?

**Review by alallema — APPROVED** [review]
LGTM!
Beautiful cleaning job

File: README.md [hunk 3]
```diff
@@ -19,23 +19,38 @@
...
 it, add the following line to your Podfile:
 
-```ruby
+```bash
```
Comment (curquiza) [inline README.md:53]:
```suggestion
```ruby
```

Comment (curquiza) [reply README.md:53]: Podfile format is in Ruby 😁

**Review by curquiza — DISMISSED** [review]

**Review by alallema — APPROVED** [review]
Well done!

**Comment by curquiza** [comment]
You have to merge https://github.com/meilisearch/meilisearch-swift/pull/181 before :)

**Comment by bidoubiwa** [comment]
bors merge

**Review by alallema — APPROVED** [review]
LGTM! 🎉

**Comment by bidoubiwa** [comment]
bors merge

**Pull request merged**